lib/buffy/throttle.ex

```diff
@@ -219,6 +223,16 @@ defmodule Buffy.Throttle do
       @impl GenServer
       @spec handle_info(:timeout, Buffy.Throttle.state()) :: {:stop, :normal, Buffy.Throttle.state()}
       def handle_info(:timeout, {key, args}) do
+        selected_jitter = min(:rand.uniform(unquote(jitter) + 1) - 1, 0)
```

Comment:
Boy do I feel dumb asking this question: wouldn't `min/2` with `0` as the second argument always return `0`, thus setting the selected jitter consistently to `0`?

Reply:
For how simple `min/2` and `max/2` are, I always feel like they trip me up when I read them 🤷🏻